src/graphics/text.rs

```diff
@@ -64,6 +64,15 @@ impl Font {
     {
         VectorFontBuilder::new(path)?.with_size(ctx, size)
     }
+    /// Creates a `Font` from a slice of binary data.
+    ///
+    /// TrueType and OpenType fonts are supported.
+    ///
+    /// This is useful in combination with [`include_bytes`](std::include_bytes), as it
+    /// allows you to include your audio data directly in the binary.
+    pub fn from_file_data(ctx: &mut Context, data: &'static [u8], size: f32) -> Result<Font> {
```

Comment:
Could this be named something like `from_vector_data` or `from_vector_file_data`? This interface won't work for bitmap fonts (as you can't specify a `size` for those, and they're made up of more than one file), so I want to make sure that we're not setting ourselves up for confusion!

Also, any function that interacts with TTF files needs to be marked with `#[cfg(feature = "font_ttf")]`, as users can disable TTF support - I think that's why the build is failing.